spec/views/insured/families/_qles_carousel.html.erb_spec.rb

```diff
@@ -60,6 +60,7 @@
       assign(:multiroles, true)
       assign(:person, person)
       allow(person).to receive(:active_employee_roles).and_return([employee_role])
+      allow(view).to receive(:individual_market_is_enabled?).and_return(false)
```

Comment:
Can you modify this so we have test cases confirming both situations.

If individual_market_is_enabled? => true we should see both sets of options
if individual_market_is_enabled? => false, we should see only the one.